view-admin/src/pages/postPrizes/index.tsx

```diff
@@ -26,13 +26,12 @@ const Page: NextPage = () => {
   );
   const [prizeNameJp, setPrizeNameJp] = useState<string>("");
   const [prizeNameEn, setPrizeNameEn] = useState<string>("");
-  const fileInputRef = useRef<HTMLInputElement>(null);
-  const [imageFile, setImageFile] = useState<File | null>(null);
+  const fileInputRef = useRef<HTMLInputElement>();
```

Comment:
`null` 入れないとエラー出ませんか？
```suggestion
  const fileInputRef = useRef<HTMLInputElement>(null);
```